contrib/python/api/skydive/rest/client.py

```diff
@@ -167,3 +167,33 @@ def edgerule_list(self):
     def edgerule_delete(self, rule_id):
         path = "/api/edgerule/%s" % rule_id
         self.request(path, method="DELETE")
+
+    def injection_create(self, srcQuery="", dstQuery="", type="icmp4",
```

Comment:
Python typically uses lowercase_underscore_delimited rather than camelCase for variables so renamed:

for example replaced `srcQuery` with `src_query`, etc.